### PR TITLE
Feature/fix metadata

### PR DESCRIPTION
**Pull request by diba1013**
*Before submission, please check that ...*

- [x] the code follows the [Clean Code](https://clean-code-developer.com/) guidelines.
- [x] the necessary tests are either created or updated.
- [x] all status checks (Travis CI, SonarCloud, etc.) pass.
- [x] your commit history is cleaned up.
- [x] ~you updated the changelog.~
- [x] ~you updated necessary documentation within [retest/docs](https://github.com/retest/docs).~

## Description

> TL;DR Fix some Kryo (de)serialization issues and reversed parameters

### CODE REVIEW

File: src/main/java/de/retest/recheck/ui/diff/meta/MetadataDifferenceFinder.java [hunk 1]
```diff
@@ -10,24 +10,24 @@
 
 public class MetadataDifferenceFinder {
 
-	public MetadataDifference findDifferences( final SutState expected, final SutState actual ) {
-		return findDifferences( expected.getMetadata(), actual.getMetadata() );
+	public MetadataDifference findDifferences( final SutState actual, final SutState expected ) {
```
Comment (roesslerj) [inline src/main/java/de/retest/recheck/ui/diff/meta/MetadataDifferenceFinder.java]: Everywhere else, it is `expected, actual` ... I would keep that order to be consistent.

**Review by roesslerj — CHANGES_REQUESTED** [review]

File: src/main/java/de/retest/recheck/ui/diff/meta/MetadataDifferenceFinder.java (hunk 1 above)
Comment (diba1013) [reply src/main/java/de/retest/recheck/ui/diff/meta/MetadataDifferenceFinder.java]: Well, that's embarrassing... 
`RecheckDifferenceFinder#findDifferences` is changed, therefore I assumed this was mixed, too.

Will revert that

Comment (roesslerj) [reply src/main/java/de/retest/recheck/ui/diff/meta/MetadataDifferenceFinder.java]: Oh. We should fix `RecheckDifferenceFinder#findDifferences` as well...

Comment (diba1013) [reply src/main/java/de/retest/recheck/ui/diff/meta/MetadataDifferenceFinder.java]: Done

**Review by roesslerj — APPROVED** [review]

**Pull request merged**